corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -205,7 +211,10 @@ hqDefine("cloudcare/js/formplayer/menus/api", function () {
         return API.queryFormplayer(options, "get_endpoint");
     });
 
-    FormplayerFrontend.getChannel().reply("entity:get:details", function (options, isPersistent, isShortDetail, isRefreshCaseSearch) {
+    FormplayerFrontend.getChannel().reply("entity:get:details", function (options,
+        isPersistent,
+        isShortDetail,
+        isRefreshCaseSearch) {
```

Comment:
Nit: same with new lines here, is this is new style guideline?

Reply:
I'll fix this. It's left over from when I was using an additional param that took the line over the length limit.

Reply:
https://github.com/dimagi/commcare-hq/pull/33802/commits/bd4f507e521baec747e527a524ab2692904c326b